package-lock.json

```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "next-auth",
```

Comment:
What happened here? Could we revert this change?

Reply:
We can, I just ran npm install, sorry.